### PR TITLE
Upgrade to gazebo 8+

### PR DESCRIPTION
**Pull request by mateus-amarante**
## Feature

* Upgrade the plugin to Gazebo 8+ while keeping the compatibility with version 7 (my main target is Gazebo 9, but I think it will also work for newer versions)
* Setup ros-action-ci GitHub Action workflow

@ivan1993br, I would appreciate your review :smile:

### CODE REVIEW

**Comment by mateus-amarante** [comment]
The tests are passing now

**Review by caioaamaral — APPROVED** [review]
LGTM

File: deps.repos [hunk 1]
```diff
@@ -0,0 +1,5 @@
+repositories:
```
Comment (ronaldoevangelista) [inline deps.repos:1]: This dependency shouldn't be in the `package_set`?

**Review by ronaldoevangelista — APPROVED** [review]

File: deps.repos (hunk 1 above)
Comment (mateus-amarante) [reply deps.repos:1]: It is properly set in BIR's private environment. This is a public repository, so I'm relying on tools commonly used by the ROS community. In this case, I'm providing a [vcstool](https://github.com/dirk-thomas/vcstool) description to be used by the [action-ros-ci](https://github.com/ros-tooling/action-ros-ci) to get not-released dependencies source-code (in this case, the `sonar_msgs` package).

**Pull request merged**